common/src/main/java/net/vyhub/abstractClasses/AUser.java

```diff
@@ -13,34 +13,26 @@
 import java.util.logging.Level;
 
 import static java.util.logging.Level.INFO;
-import static net.vyhub.lib.Utility.checkResponse;
 
-public abstract class AUser {
+public abstract class AUser extends SuperClass {
     public static Map<String, VyHubUser> vyHubPlayers = new HashMap<>();
-
-    private final VyHubPlatform platform;
-
     private static Semaphore userCreateSem = new Semaphore(1, true);
 
     public AUser(VyHubPlatform platform) {
-        this.platform = platform;
-    }
-
-    public VyHubPlatform getPlatform() {
-        return platform;
+        super(platform);
     }
 
     public void checkPlayerExists(String playerId, String playerName) {
         VyHubUser user = getUser(playerId);
 
         if (user == null) {
-            platform.log(Level.WARNING, String.format("Could not register player %s, trying again in a minute..", playerName));
+            getPlatform().log(Level.WARNING, String.format("Could not register player %s, trying again in a minute..", playerName));
```

Comment:
Seems a little bit unclean to always call `getPlatform()` within the same class.
Make platform `protected` and access it directly instead.